src/test/java/se/sundsvall/supportmanagement/integration/db/model/ErrandEntityTest.java

```diff
@@ -73,7 +73,7 @@ void hasValidBuilderMethods() {
 			.withCategoryTag(categoryTag)
 			.withNamespace(namespace)
 			.withCreated(created)
-			.withCustomer(customer)
+			.withStakeholders(List.of(stakeholder))
```

Comment:
Verify handling of stakeholders in OnCreate and OnUpdate methods?

Reply:
Fixed